terraform-mode.el

```diff
@@ -252,18 +252,54 @@
       (when (re-search-forward (concat "/\\(.*?\\)/" provider "\\]") nil t)
         (match-string 1)))))
 
+(defun terraform--get-resource-provider-source (provider &optional dir)
+  "Return provider source for PROVIDER located in DIR."
```

Comment:
It's not clear what the source is.  Is it buffers, files, file names?

Reply:
ok, I'll add these details in the doc.